### ISSUE TITLE
Can I synchronize multiple calendars independently?

### ISSUE THREAD

**Issue by kallekruse** [39aldo39/Radicale-DecSync#28]
Use case: I would like to sync two calendars:

1. My personal calendar, between my Ubuntu laptop and my Android phone. This already works using Radicale-DecSync, DecSyncCC and SyncThing (kudos!)
2. A shared calendar, between my laptop and multiple Android phones, maybe also another Ubuntu laptop.

I would like to share these two calendars independently: I would not want to give all devices who will have access to the shared calendar also access to my personal calendar. Is this doable with Radicale-DecSync, DecSyncCC and SyncThing?

**Comment by Mugen36** [comment]
Yes, you can. In the DecSync app you select the Calendars you wish to sync. However, the prerequisit here is that you trust the other devices respectively the owner of the other devices as one can change the calendars to sync with a single click.

**Comment by kallekruse** [comment]
Thanks, @Mugen36 . So there is no way to share calendars independently without having to trust all clients of all calendars?

**Comment by Mugen36** [comment]
> Thanks, @Mugen36 . So there is no way to share calendars independently without having to trust all clients of all calendars?

Well, when you are using Syncthing for example, you are syncing the whole Decsync folder. So no matter the implementation inside the Decsync app, so every client who has access to the folder can have a look at it.

If you want a calendar sync solution for multiple users, you probably should look into something like Nextcloud.
Although, I guess, you could run theoretically run multiple instances of Decsync.

**Comment by 39aldo39** [comment]
It is (almost) possible to do this by using two separate DecSync directories: one personal and one shared one. The personal one you only share between your own devices while the shared one you also share with other people.
- In DecSync CC it is possible to add an additional DecSync directory so you can have both.
- In Evolution DecSync you have to select the directory anyway when adding a calendar, so this is also supported.
- In Radicale DecSync, however, this is not yet possible as you can only select one directory. It might be possible to run multiple instances based on different configs, but I'm not sure that works.

**Comment by 39aldo39** [comment]
It is not, but mainly because it is technically difficult as the DecSync directory is stored is in the config file which is the same for all Radicale users.

**Comment by kallekruse** [comment]
I finally tried running two Radicale instances with separate config files in parallel for syncing calendars separately. It seems to work in general (so far only tested syncing one Ubuntu laptop with one Android phone), but I am running into some problems with the new Radicale instance which I have never observed with the first one:

1. In Thunderbird each event is shown twice in the event list.
2. Events created on Android and later removed on Ubuntu/Thunderbird are not automatically removed on Android/Etar. Creating on Android and removing on Ubuntu works though.

I cannot see how these issues would be caused by my double-Radicale setup. Neither can I spot any relevant-looking difference between the two configurations (first vs second Radicale instance plus the corresponding SyncThing directories and DecSync-CC directories).

**Comment by kallekruse** [comment]
Another question: Does anything speak against syncing the profile subfolders of the `calendars` folder (e.g. `<decsync directory>/calendars/742c...f0e`) separately? Then I could choose which calendar to share with whom via SyncThing.

**Comment by JohnVeness** [comment]
This is an old issue, but I was wondering if you've had more experience with this, @kallekruse ? I have been using DecSync-Radicale/DecSync-CC happily for many years with a single folder shared via Syncthing, but I too am now looking to have a new shared calendar with someone and don't want them able to access my private calendar.
